### PR TITLE
fix: incoming call not vibrating in vibrate mode (AR-2618)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2618" title="AR-2618" target="_blank">BugAR-2618</a>  Incoming call rings when ringing is disabled
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

When device is on vibrate mode, incoming call is not vibrating

### Causes (Optional)

Not implemented

### Solutions

Use the vibrator to vibrate an incoming call when device mode is vibrate.

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/media/CallRinger.kt [hunk 1]
```diff
@@ -35,12 +51,39 @@ class CallRinger @Inject constructor(private val context: Context) {
...
+            } else {
+                appLogger.i("Skipping vibration");
+            }
+        }
+    }
+
     fun stop() {
+        appLogger.i("Stopping ringing");
         try {
+            vibrator?.cancel()
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/media/CallRinger.kt:86]: Is there any option to `release`/`destroy` the vibrator after it is being used? Not here necessarily but just so we don't have it leaked? 

Just a question, I have no idea what `cancel` actually does

**Review by alexandreferris — APPROVED** [review]
Great work! I just have a tiny question, more like to understand what `cancel` does

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1179/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5025282)(5025282)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5025282)(5025282)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/media/CallRinger.kt (hunk 1 above)
Comment (ohassine) [reply app/src/main/kotlin/com/wire/android/media/CallRinger.kt:86]: No we can only cancel it.
cancel will turn the vibrator off

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1179/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5032914)(5032914)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5032914)(5032914)-dev-release.apk)

**Pull request merged**